src/core_modules/capture-core/components/Pages/NewRelationship/RegisterTei/RegisterTei.types.js

```diff
@@ -17,4 +14,4 @@ export type OwnProps = {|
     onSave: Function,
```

Comment:
A bit confused with this `onSave` function I see in some places we are changing the annotation of this function to `() => void`. However on the `RegisterTei.component.js` line 85 I can see that the function take 2 arguments. 

Are these two different onSave functions? If they are different perhaps we can find a way to make this more clear (?) 

If they arent different perhaps we could be consistent about its annotation?